angular-ngrx-scss/src/app/repository/components/repo-header/repo-navigation/repo-navigation.component.scss

```diff
@@ -0,0 +1,63 @@
+@use '../../../../shared/styles/variables.scss';
+
+.container {
+  max-width: 1536px;
+  margin: 0 auto 0 0;
+  nav {
+    margin-bottom: -1px;
+    display: flex;
+    ul {
+      list-style: none;
+      padding: 0;
+      margin: 0;
+
+      a {
```

Comment:
Suggestion: use the ".tab" class here and a modifier ".tab--active" / ".tab--inactive" for the states.